src/python/WMCore/ACDC/CouchFileset.py

```diff
@@ -99,9 +99,22 @@ def add(self, files, mask = None):
         _add_
 
         Add files to this fileset
+
+        Note: if job was lumi based splitted, then we do not have
+        reliable events information. If job was event based splitted,
+        then we do not have reliable lumi information.
         """
         filteredFiles = []
         if mask:
+            for f in files:
+                # There is no LastEvent for last job of a file
+                if mask['LastEvent'] and mask['FirstEvent']:
+                    f['events'] = mask['LastEvent'] - mask['FirstEvent']
```

Comment:
What is the event to recover in the example?
Are those events inclusive here? 
'LastEvent': 163001L, 'FirstEvent': 162001L,
In that case, shouldn't +1?

In case of below,  event to be recovered is 1 or 0?
'LastEvent': 1, 'FirstEvent': 1,